src/main/java/com/clussmanproductions/trafficcontrol/event/EntityClassRenderer.java

```diff
@@ -54,6 +57,14 @@ public static void keyPress(KeyInputEvent e)
 		if (ClientProxy.entityClassRendererKey.isPressed())
 		{
 			performEntityClassRender = !performEntityClassRender;
+			if(performEntityClassRender)
```

Comment:
There's probably a better way to do this, but it works so meh lmao

Reply:
You could do something like this:
```java
performEntityClassRender = !performEntityClassRender;
Minecraft.getMinecraft().player.sendMessage(new TextComponentString(TextFormatting.YELLOW + "[" + ModTrafficControl.MODNAME + "] Entity Class Viewer " + (performEntityClassRender ? "Enabled" : "Disabled")));
```
This would condense it into two lines and remove duplication. 

Also I should mention if anything is wrong there, I apologize, I'm not a forge developer, I code more spigot plugins but this would be my solution to this.

Reply:
Change implemented in https://github.com/CSX8600/trafficcontrol/pull/79/commits/000f622beb609e1e7e5b09505bece9383f07d554